sound/soc/sof/ipc4-priv.h

```diff
@@ -66,6 +66,8 @@ struct sof_ipc4_fw_library {
  * @nhlt: NHLT table either from the BIOS or the topology manifest
  * @mtrace_type: mtrace type supported on the booted platform
  * @mtrace_log_bytes: log bytes as reported by the firmware via fw_config reply
+ * @num_playback_streams: max number of playback DMAs, needed for CHAIN_DMA offset
```

Comment:
nitpicks:
1. commit message:
```
- The CHAIN_DMA IPC needs the number of playback streams as an offset for the dma_id.
+ The CHAIN_DMA IPC needs the number of playback streams as a start offset for the dma_id of a capture stream.
```
2. Is this documented in any way for the IPC4 protocol? We just need to be clear on this for other vendors who will try to implement ChainDMA.

Reply:
updated commit message as suggested.

I also added a note 
"
The CHAIN_DMA IPC is currently only functional on Intel HDaudio DMAs,
    and gated by the snd_sof_is_chain_dma_supported() helper.
"